admin_tests/nodes/test_views.py

```diff
@@ -442,3 +443,35 @@ def test_restart_stuck_registration(self):
         view.post(self.request)
 
         nt.assert_equal(self.registration.archive_job.status, u'SUCCESS')
+
+
+class TestRemoveStuckRegistrationsView(AdminTestCase):
+    def setUp(self):
+        super(TestRemoveStuckRegistrationsView, self).setUp()
+        self.user = AuthUserFactory()
+        self.registration = RegistrationFactory(creator=self.user)
+        self.registration.save()
+        self.view = RemoveStuckRegistrationsView
+        self.request = RequestFactory().post('/fake_path')
+
+    def test_get_object(self):
+        view = RemoveStuckRegistrationsView()
+        view = setup_log_view(view, self.request, guid=self.registration._id)
+
+        nt.assert_true(self.registration, view.get_object())
+
+    def test_remove_stuck_registration(self):
+        view = RemoveStuckRegistrationsView()
+        view = setup_log_view(view, self.request, guid=self.registration._id)
+        from django.contrib.messages.storage.fallback import FallbackStorage
```

Comment:
Imports should go at the top of the file unless circumstances (e.g. circular imports) won't allow it.

Reply:
This creates a strange circular import where the admin app hangs on start up and never fully loads